Form/Extension/LayoutFormTypeExtension.php

```diff
@@ -65,9 +67,9 @@ public function buildView(FormView $view, FormInterface $form, array $options)
 
     public function finishView(FormView $view, FormInterface $form, array $options)
     {
-        if (!$view->parent && $options['compound'] && $view->vars['horizontal']) {
+        if (!$view->parent && $options['compound'] && $view->vars['layout']) {
             $class = isset($view->vars['attr']['class']) ? $view->vars['attr']['class'].' ' : '';
-            $view->vars['attr']['class'] = $class.'form-horizontal';
+            $view->vars['attr']['class'] = $class.'form-'.$view->vars['layout'];
```

Comment:
Scratch that, I missed the check above it, if it's null or false then this if statement doesn't even make it in.